### DISCUSSION TITLE
How do I get a Galaxy Brain badge if I have answered 2 discussions?

### DISCUSSION THREAD

**Discussion by AthulOntop** [community/community#19986] [Other Feature Feedback, Questions, & Ideas] [answered]
s

**Comment by DxstyLy** [comment, marked as answer]
Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! :)
